### ISSUE TITLE
Code elements need special cases

### ISSUE THREAD

**Issue by lukasoppermann** [worddrop/bookworm#10]
We need to define special cases for code, as for e.g. an SVG can lead to extremely long reading times while nobody actually reads elements like this:

``` html
<path d="M28.4 40.2C28.4 34.5 33 30 38.7 30c3.7 0 7 2 8.7 5 .6-.5 1.7-1 3.4-1.5-2.4-4.3-7-7.2-12.2-7.2-7.7 0-14 6.2-14 14 0 1.5.4 3 1 4.5.4-.5 1.4-1.3 3-2l-.2-2.6z" fill="#FFC300"/>
```

So maybe we should base the estimate for code on tags?

**Comment by jeroenvisser101** [comment]
What about stripping HTML tags? It seems that solves a lot. Also, maybe that shouldn't be part of the library, what do you think?

**Comment by lukasoppermann** [comment]
Well, I would strip tags only outside of `<code>` blocks using php_strip_tags. Inside the `<code>` block we should try to estimate the because if you write an article with much code, it does take quite a bit to read it, so if we strip it, it would be a very wrong estimate.

**Comment by lukasoppermann** [comment]
Fixed when #14 is merged.
